docs/pages/api-reference/operators/window.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Window
+order: 0
+status: published
+---
+### Window
+
+<Divider>
+<LeftSection>
+Operator to do aggregation of events into windows based on their timestamps. Window operator must always be preceded by a [groupby](/api-reference/operators#groupby) operator.
+
+#### Parameters
+
+<Expandable title="type" type="Window">
+
+The type of window to group event on. Possible values are `"hopping"`, `"tumbling"`, `"session"`.
+
+`"hopping"`: Take in two parameters `"duration"` and `"stride"`, the `"duration"` parameters indicate how long the window is while `"gap"` indicate how far apart the window should be. Hopping window can overlap meaning one event can belong to two different windows. `"stride"` should be smaller than `"duration"`.
```

Comment:
This is double documentation - would instead suggest to mention something simpler here e.g. "If window is "hopping", `duration` and `stride` must be set. ....

And then do the acutal documentation of all these params in their own expandable section.